Wiki-Become-A-Mod.md

```diff
@@ -1,6 +1,6 @@
 # Requirements to become a wiki Moderator
 
-There are many ways to contribute to the [wiki](Wiki) without being part of the [Senior Group](Senior-Wiki-Mods). However, if you want to become one, there are certain requirements that you must meet first:
+There are many ways to contribute to the [wiki](Wiki) without being part of the [Senior Group](Wiki-Senior-Mods). However, if you want to become one, there are certain requirements that you must meet first:
```

Comment:
I couldn't find any file named Wiki-Senior-Mods.md in the Wiki, nor in this commit. What does this link point to?